src/BuildScriptGenerator/Ruby/RubyBashBuildSnippetProperties.cs

```diff
@@ -17,5 +17,7 @@ internal class RubyBashBuildSnippetProperties
         public bool GemfileExists { get; set; }
 
         public bool ConfigYmlFileExists { get; set; }
+
+        public string CustomBuildCommand { get; set; }
```

Comment:
is it an environment variable or a property that will be passed to Oryx? I'm bit confused now ... according to the test it's an environment variable .. so do we really need it as buildsnippet properties?

Reply:
It is passed in as env variable, and then as options (Hierarchy). And then we need to checking this property for generating the buildSnippet so we need to include it into the `RubyBashBuildSnippetProperties.cs` file.